.grit/patterns/js/curly.md

```diff
@@ -0,0 +1,69 @@
+---
+title: Enforces braces for if/for/do/while statements.
+tags: [good-practice]
+---
+
```

Comment:
You should include a description for rules from the stdlib. Ex:

```suggestion
This matches the [eslint rule](https://eslint.org/docs/latest/rules/curly).
```

Reply:
@morgante done